ngraph/core/include/openvino/opsets/opset2_tbl.hpp

```diff
@@ -2,142 +2,142 @@
+_OPENVINO_OP_REG(ReorgYolo, ov::op::v0)  // Missing in opset1
 
-OPENVINO_OP(Reshape, ov::op::v1)
-OPENVINO_OP(Result, ov::op::v0)
+_OPENVINO_OP_REG(Reshape, ov::op::v1)
+_OPENVINO_OP_REG(Result, ov::op::v0)
 
 // Moved out of opset2, it was added to opset1 by mistake
-// OPENVINO_OP(Reverse, ov::op::v1)
+// _OPENVINO_OP_REG(Reverse, ov::op::v1)
 
-OPENVINO_OP(ReverseSequence, ov::op::v0)
+_OPENVINO_OP_REG(ReverseSequence, ov::op::v0)
 
 // Moved out of opset2, it was added to opset1 by mistake
-// OPENVINO_OP(RNNCell, ov::op::v0)
+// _OPENVINO_OP_REG(RNNCell, ov::op::v0)
 
-OPENVINO_OP(ROIPooling, ov::op::v0)  // Missing in opset1
+_OPENVINO_OP_REG(ROIPooling, ov::op::v0)  // Missing in opset1
 
-OPENVINO_OP(Select, ov::op::v1)
-OPENVINO_OP(Selu, ov::op::v0)
-OPENVINO_OP(ShapeOf, ov::op::v0)
+_OPENVINO_OP_REG(Select, ov::op::v1)
+_OPENVINO_OP_REG(Selu, ov::op::v0)
+_OPENVINO_OP_REG(ShapeOf, ov::op::v0)
 
 // Moved out of opset2, it was added to opset1 by mistake
-// OPENVINO_OP(ShuffleChannels, ov::op::v0)
+// _OPENVINO_OP_REG(ShuffleChannels, ov::op::v0)
 
-OPENVINO_OP(Sign, ov::op::v0)
-OPENVINO_OP(Sigmoid, ov::op::v0)
-OPENVINO_OP(Sin, ov::op::v0)
-OPENVINO_OP(Sinh, ov::op::v0)
-OPENVINO_OP(Softmax, ov::op::v1)
-OPENVINO_OP(Sqrt, ov::op::v0)
-OPENVINO_OP(SpaceToDepth, ov::op::v0)
-OPENVINO_OP(Split, ov::op::v1)
-OPENVINO_OP(SquaredDifference, ov::op::v0)
-OPENVINO_OP(Squeeze, ov::op::v0)
-OPENVINO_OP(StridedSlice, ov::op::v1)
-OPENVINO_OP(Subtract, ov::op::v1)
-OPENVINO_OP(Tan, ov::op::v0)
-OPENVINO_OP(Tanh, ov::op::v0)
-OPENVINO_OP(TensorIterator, ov::op::v0)
-OPENVINO_OP(Tile, ov::op::v0)
-OPENVINO_OP(TopK, ov::op::v1)
-OPENVINO_OP(Transpose, ov::op::v1)
-OPENVINO_OP(Unsqueeze, ov::op::v0)
-OPENVINO_OP(VariadicSplit, ov::op::v1)
+_OPENVINO_OP_REG(Sign, ov::op::v0)
+_OPENVINO_OP_REG(Sigmoid, ov::op::v0)
+_OPENVINO_OP_REG(Sin, ov::op::v0)
+_OPENVINO_OP_REG(Sinh, ov::op::v0)
+_OPENVINO_OP_REG(Softmax, ov::op::v1)
+_OPENVINO_OP_REG(Sqrt, ov::op::v0)
+_OPENVINO_OP_REG(SpaceToDepth, ov::op::v0)
+_OPENVINO_OP_REG(Split, ov::op::v1)
+_OPENVINO_OP_REG(SquaredDifference, ov::op::v0)
+_OPENVINO_OP_REG(Squeeze, ov::op::v0)
+_OPENVINO_OP_REG(StridedSlice, ov::op::v1)
+_OPENVINO_OP_REG(Subtract, ov::op::v1)
+_OPENVINO_OP_REG(Tan, ov::op::v0)
+_OPENVINO_OP_REG(Tanh, ov::op::v0)
+_OPENVINO_OP_REG(TensorIterator, ov::op::v0)
+_OPENVINO_OP_REG(Tile, ov::op::v0)
+_OPENVINO_OP_REG(TopK, ov::op::v1)
+_OPENVINO_OP_REG(Transpose, ov::op::v1)
+_OPENVINO_OP_REG(Unsqueeze, ov::op::v0)
+_OPENVINO_OP_REG(VariadicSplit, ov::op::v1)
 
 // Moved out of opset2, it was added to opset1 by mistake
-// OPENVINO_OP(Xor, ov::op::v0)
+// _OPENVINO_OP_REG(Xor, ov::op::v0)
 
 // New operations added in opset2
-OPENVINO_OP(Gelu, ov::op::v0)
-OPENVINO_OP(BatchToSpace, ov::op::v1)
-OPENVINO_OP(SpaceToBatch, ov::op::v1)
+_OPENVINO_OP_REG(Gelu, ov::op::v0)
+_OPENVINO_OP_REG(BatchToSpace, ov::op::v1)
+_OPENVINO_OP_REG(SpaceToBatch, ov::op::v1)
```

Comment:
why do we need this change?

Reply:
Because we introduce macro `OPENVINO_OP` in order to avoid redefinition, I have changed this internal macro.